src/nvim/lua/vim.lua

```diff
@@ -424,7 +424,7 @@ end
 --- Without a runtime, writes to :Messages
 ---@see :help nvim_notify
 ---@param msg string Content of the notification to show to the user
----@param log_level number|nil enum from vim.log.levels
+---@param log_level number|nil enum from |vim.log.levels|
```

Comment:
I meant this line will also be overwritten. You need to change line 427 in `src/nvim/lua/vim.lua` as well.

Reply:
this is what i changed????

Reply:
Did you `git add` that file?

Reply:
Oops. Sorry.